src/main/java/frc/robot/RobotContainer.java

```diff
@@ -58,6 +58,12 @@ private void configureButtonBindings() {
 		JoystickButton xboxAButton = new JoystickButton(m_controller, Constants.XBOX_A);
 		xboxAButton.onTrue(new InstantCommand(m_driveTrain::toggleFieldCentric));
 
+		// button B
+		JoystickButton xboxBButton = new JoystickButton(m_controller, Constants.XBOX_LB);
+		// inline command to toggle lock mode when button B is pressed
+		xboxBButton.onTrue(new InstantCommand(m_driveTrain::toggleLockMode));
```

Comment:
Please follow the commenting convention of the other lines. It feels unnecessary to comment "Button B" when both the variable name and "XBOX" enum lists the button.

Please don't use the bumpers. Those will almost certainly be for the intake. You can use an unused A,B,X,Y or START , BACK.